lib/mutant/mutator/node/send.rb

```diff
@@ -26,8 +26,9 @@ class Send < self
           to_i:          %i[to_int],
           to_a:          %i[to_ary],
           to_h:          %i[to_hash],
-          at:            %i[fetch],
-          :[] =>         %i[at fetch],
+          at:            %i[fetch key?],
+          fetch:         %i[key?],
+          :[] =>         %i[at fetch key?],
```

Comment:
@mbj I wonder if in the future that instead of defining a hash mapping the method to possible alternatives, we just defined a list of lists where you'd list a series of methods, ranked from most to least powerful.

So in this case, instead of having entries for `#at`, `#fetch` and #[]` we could just write:

```ruby
[
  # ...
  `%i[ [] at fetch key? ]`
  # ...
]
```

Of course under the hood you could have some kind of index to make it efficient to lookup a specific method and find less powerful alternatives, but it would eliminate duplicate information like we have now.